instructionAPI/src/InstructionDecoder-x86.C

```diff
@@ -1831,10 +1841,15 @@ namespace Dyninst
 
         if (decodedInstruction->getEntry()->getID() == e_ret_near ||
             decodedInstruction->getEntry()->getID() == e_ret_far) {
-           Expression::Ptr ret_addr = makeDereferenceExpression(makeRegisterExpression(is64BitMode ? x86_64::rsp : x86::esp),
```

Comment:
undo the whitspace changes to 1834-1837 (or 1844-1847 in new code), so there isn't a diff for these lines

Reply:
Done